CONTRIBUTING.md

```diff
@@ -81,15 +81,21 @@ Ready to contribute? Here's how to set up `cpp2wasm` for local development.
     make entangle
     ```
 
-7. Commit your changes and push your branch to GitHub::
+7. When `src/js/newtonraphsonwasm.js` has changed, make sure the WebAssembly module is also rebuild with (requires [emscripten](README.md#accessing-c-function-from-JavaScript-in-web-browser) to run).
```

Comment:
```suggestion
7. When `src/js/newtonraphsonwasm.js` changes, WebAssembly module also has to be rebuilt. This will require [emscripten](README.md#accessing-c-function-from-JavaScript-in-web-browser). To rebuild the WebAssembly module run:
```